src/backend/drm/device/mod.rs

```diff
@@ -14,6 +14,8 @@ use nix::sys::stat::fstat;
 
 pub(super) mod atomic;
 pub(super) mod legacy;
+use crate::utils::{Physical, Size};
```

Comment:
I notice the rest of the file uses the qualified path for internal crates rather than use. Not sure why exactly...